src/lib/oauth2-model.ts

```diff
@@ -188,6 +190,13 @@ export class OAuth2Model implements RefreshTokenModel {
      * Get user.
      */
     getUser = async (username: string, password: string): Promise<User | Falsey> => {
+        if (password === SSO_PASSWORD) {
```

Comment:
I haven't found a clean solution while sticking to the `oauth2-server` module, hence I used this approach which I don't think is perfect at all.